### ISSUE TITLE
Add cmds to move & copy existing subjects to subject sets

### ISSUE THREAD

**Issue by camallen** [zooniverse/panoptes-cli#158]
Would you support the cli to have the ability to move a subject between sets or even copy to a new set? 

Noting that subject set already has the the `add-subjects` & `remove-subjects` cmds on the subject set.

Move could could possibly wrap the above set commands but on the subject resource instead?
- `panoptes subject 1 move_to 321` : link to set 321, remove from current set 
- `panoptes subject 1 copy_to 321`. : link to set 321 and leave intact at current set.

Thoughts?

**Comment by adammcmaster** [comment]
Copying sounds the same as the existing `add-subjects` command, but we absolutely can add a move command if that's something people need.

**Comment by camallen** [comment]
yes, i agree. However i think adding the cmds above to the subject resource and we could use the `copy` and `move` terminology?E.g.
- I want to copy this subject to subject set 1
- I want to move this subject from subject set 2 to subject set 1

**Comment by adammcmaster** [comment]
Ok, yeah I see how that would be useful.
